packages/gatsby-telemetry/src/postinstall.js

```diff
@@ -1,12 +1,14 @@
 const showAnalyticsNotification = require(`./showAnalyticsNotification`)
+const isTruthy = require(`./is-truthy`)
 
 let enabled = false
 try {
   const ci = require(`ci-info`)
   const Configstore = require(`configstore`)
   const config = new Configstore(`gatsby`, {}, { globalConfigPath: true })
   enabled = config.get(`telemetry.enabled`)
-  if (enabled === undefined && !ci.isCI) {
+  const disabled = isTruthy(process.env.GATSBY_TELEMETRY_DISABLED)
```

Comment:
It would be cool to have this check live in only a single place. 
One option would be to expose the disabled field from the event store instead.

Reply:
I went ahead and did this